src/functions.php

```diff
@@ -736,3 +736,30 @@ function parse($value)
 {
     return Context::get()->getConfig()->parse($value);
 }
+
+function locateBinaryPath($name)
+{
+    $nameEscaped = escapeshellarg($name);
+
+    // Try `command`, should cover all Bourne-like shells
+    if (commandExist("command")) {
+        return run("command -v $nameEscaped")->toString();
+    }
+
+    // Try `which`, should cover most other cases
+    if (commandExist("which")) {
```

Comment:
I was thinking about keeping as much possible to previous implementation, but in command -v is common then it's okay.